systemtest/helpers.bash

```diff
@@ -263,7 +266,12 @@ start_registry() {
     AUTHDIR=$TESTDIR/auth
     mkdir -p $AUTHDIR
 
-    local -a reg_args=(-v $AUTHDIR:/auth:Z -p $port:5000)
+    if [[ "$enable_delete" == "true" ]]; then
+        local -a reg_args=(-v $AUTHDIR:/auth:Z -p $port:5000 \
+	    -e REGISTRY_STORAGE_DELETE_ENABLED=true)
+    else
+        local -a reg_args=(-v $AUTHDIR:/auth:Z -p $port:5000)
```

Comment:
The code below uses `reg_args+=` in conditionals; using it for the `enable_delete` case as well would avoid duplication of the `-v … -p …` parameters.

Reply:
sure